dataprofiler/profilers/profiler_options.py

```diff
@@ -1063,6 +1090,45 @@ def _validate_helper(self, variable_path: str = "DataLabelerOptions") -> list[st
             errors.append(f"{variable_path}.max_sample_size must be greater than 0.")
         return errors
 
+    @classmethod
+    def load_from_dict(
+        cls,
+        data,
+        options: dict | None = None,
+    ) -> BaseOption:
```

Comment:
output here should be `DataLabelerOptions`

Reply:
didn't like that ... but likes `BaseOptionT`